packages/jest-cli/src/run_jest.js

```diff
@@ -67,26 +68,75 @@ const getTestPaths = async (
   });
 };
 
+function formatWhyRunning(whyRunning) {
```

Comment:
maybe we can extract why-running-related logic to separate file?

Reply:
yeah, good call. done